app/views/imports/issues/show.html.erb

```diff
@@ -58,7 +47,13 @@
     ) do |f| %>
   <% content_for(:before_content) { f.govuk_error_summary } %>
 
-  <%= f.govuk_collection_radio_buttons :apply_changes, @form.apply_changes_options, :itself, nil %>
+  <%= f.govuk_collection_radio_buttons :apply_changes,
+                                       @form.apply_changes_options,
+                                       :itself,
+                                       ->(option) { I18n.t(option, scope: "helpers.label.import_duplicate_form.apply_changes_options.#{@existing_or_deleted}", type: @type) },
```

Comment:
If we can't use the `helpers` then we should put these locale strings somewhere else, the `helpers` are only designed to be picked up automatically by [the form builder](https://govuk-form-builder.netlify.app/building-blocks/localisation/).